social-tap/Class1.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+public class pixelCounter
+{
+	public pixelCounter()
```

Comment:
Kodėl visas kodas konstruktoriuje? Be to, šita klasė neveikiantis PixelCounter.cs duplikatas , tai ją trint reikia.